### PR TITLE
[Fix] milestone close timestamp

### LINKED ISSUE #8727: [bug] wrong close time issue on milestone
gitea version: 1.11.0+dev-56-g28f60bb5c

test url: https://gitea.com/gitea/tea/milestones?state=closed
Bildschirmfoto zu 2019-10-28 23-33-43

### PR DESCRIPTION
**Pull request by 6543**
will close #8727

### CODE REVIEW

File: models/issue_milestone.go [hunk 1]
```diff
@@ -306,7 +306,11 @@ func ChangeMilestoneStatus(m *Milestone, isClosed bool) (err error) {
 	}
 
 	m.IsClosed = isClosed
-	if _, err := sess.ID(m.ID).Cols("is_closed").Update(m); err != nil {
+	if isClosed {
+		m.ClosedDateUnix = timeutil.TimeStampNow()
+	}
```
Comment (6543) [inline models/issue_milestone.go:311]: if !isClosed value is not used so we dont ned to update value when reopen it ...

**Comment by 6543** [comment]
@techknowlogick can you add backport lable?

**Comment by 6543** [comment]
or @lunny if you have time ...

**Review by lunny — APPROVED** [review]

**Comment by lunny** [comment]
When reopen a milestone, should we remove the close time?

**Comment by 6543** [comment]
@lunny thought about this ... but it miss-use the db field + there is no usecase jet

**Review by guillep2k — APPROVED** [review]

**Comment by guillep2k** [comment]
I agree that cleaning up the `closed_date_unix` field is a gray area; expecially because we're not using null values (that's what they're for). If the field is informative and only used when `IsClosed()`, I think it can remain set with its previous value.

**Comment by 6543** [comment]
@lunny after merge I'll send backports

**Pull request merged**